Jint/Runtime/TypeConverter.cs

```diff
@@ -494,6 +495,8 @@ public static void CheckObjectCoercible(Engine engine, JsValue o)
             }
         }
 
+        private static readonly Engine _engine = new Engine();
```

Comment:
you should add new parameter `Engine engine` and pass `_engine`from the current call sites. It should be the same engine that is processing request.

Reply:
Too easy, done :-)